src/diffusers/testing_utils.py

```diff
@@ -2,13 +2,17 @@
 import random
 import unittest
 from distutils.util import strtobool
+from packaging import version
 
 import torch
 
 
 global_rng = random.Random()
 torch_device = "cuda" if torch.cuda.is_available() else "cpu"
-torch_device = "mps" if torch.backends.mps.is_available() else torch_device
+is_torch_higher_equal_than_1_12 = version.parse(version.parse(torch.__version__).base_version) >= version.parse("1.12")
+
+if is_torch_higher_equal_than_1_12:
```

Comment:
cc @pcuenca 

We need to be careful when using features that are quite new in PyTorch. This would have broken PyTorch 1.11 here. 
In the future we should run backwards compatible version tests starting from our oldest supported version. For now we need to be a bit careful when adding very new features to not break PyTorch 1.11